UnitsNet.Tests/CustomCode/LengthTests.FeetInches.cs

```diff
@@ -29,63 +31,93 @@ public void FeetInchesRoundTrip()
+            new object[]{"1′", 1, new CultureInfo("en-US")},                    // Feet only
+            new object[]{"1,000′", 1000, new CultureInfo("en-US")},             // Feet only, with seperator
+            new object[]{"1\"", 0.08333333, new CultureInfo("en-US")},          // Inches only
+            new object[]{"1″", 0.08333333, new CultureInfo("en-US")},           // Inches only
+            new object[]{"0' 1\"", 0.08333333, new CultureInfo("en-US")},       // Inches only
+            new object[]{"0' 1″", 0.08333333, new CultureInfo("en-US")},        // Inches only
+            new object[]{"0′ 1\"", 0.08333333, new CultureInfo("en-US")},       // Inches only
+            new object[]{"0′ 1″", 0.08333333, new CultureInfo("en-US")},        // Inches only
+            new object[]{"1' 1\"", 1.08333333, new CultureInfo("en-US")},       // Normal form
+            new object[]{"1′ 1″", 1.08333333, new CultureInfo("en-US")},        // Normal form
+            new object[]{" 1′ 1″ ", 1.08333333, new CultureInfo("en-US")},      // Normal form, requires trimming
+            new object[]{"1'1\"", 1.08333333, new CultureInfo("en-US")},        // Without space
+            new object[]{"1′1″", 1.08333333, new CultureInfo("en-US")},         // Without space
+            new object[]{"1 ft 1 in", 1.08333333, new CultureInfo("en-US")},
+            new object[]{"1ft 1in", 1.08333333, new CultureInfo("en-US")},
+            new object[]{"-1'", -1, new CultureInfo("en-US")},                  // Feet only
+            new object[]{"-1′", -1, new CultureInfo("en-US")},                  // Feet only
+            new object[]{"-1\"", -0.08333333, new CultureInfo("en-US")},        // Inches only
+            new object[]{"-1″", -0.08333333, new CultureInfo("en-US")},         // Inches only
+            new object[]{"-0' 1\"", -0.08333333, new CultureInfo("en-US")},     // Inches only
+            new object[]{"-0' 1″", -0.08333333, new CultureInfo("en-US")},      // Inches only
+            new object[]{"-0′ 1\"", -0.08333333, new CultureInfo("en-US")},     // Inches only
+            new object[]{"-0′ 1″", -0.08333333, new CultureInfo("en-US")},      // Inches only
+            new object[]{"-1' 1\"", -1.08333333, new CultureInfo("en-US")},     // Normal form
+            new object[]{"-1′ 1″", -1.08333333, new CultureInfo("en-US")},      // Normal form
+            new object[]{" -1′ 1″ ", -1.08333333, new CultureInfo("en-US")},    // Normal form, requires trimming
+            new object[]{"-1'1\"", -1.08333333, new CultureInfo("en-US")},      // Without space
+            new object[]{"-1′1″", -1.08333333, new CultureInfo("en-US")},       // Without space
+            new object[]{"-1 ft 1 in", -1.08333333, new CultureInfo("en-US")},
+            new object[]{"-1ft 1in", -1.08333333, new CultureInfo("en-US")},
+            new object[]{"1’000′", 1000, new CultureInfo("de-CH")},             // Feet only, with seperator
```

Comment:
I think @pgrawehr made a good point about the need to use `new CultureInfo("de-CH", false)` to avoid using any local user overrides in the machine that runs these tests. Don't expect appveyor to have this problem, but probably a good practice.

https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.-ctor?view=netcore-3.1#System_Globalization_CultureInfo__ctor_System_String_

Reply:
Good call. I added in #804